src/Evolution/Systems/RadiationTransport/M1Grey/System.hpp

```diff
@@ -0,0 +1,87 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Conservative/ConservativeDuDt.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/Fluxes.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/Sources.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Tags {
+template <class>
+class Variables;
+}  // namespace Tags
+
+/// \ingroup EvolutionSystemsGroup
+/// \brief Items related to general relativistic radiation transport
+namespace RadiationTransport {
+/// The M1 scheme for radiation transport
+///
+/// References:
+/// - Post-merger evolution of a neutron star-black hole binary with
+///   neutrino transport, Phys. Rev. D 91, 124021 (2015)
+///   arXiv:1502.04146
+namespace M1Grey {
+
+template <typename... NeutrinoSpecies>
+struct System {
+  static constexpr bool is_in_flux_conservative_form = true;
+  static constexpr bool has_primitive_and_conservative_vars = false;
+  static constexpr size_t volume_dim = 3;
+  static constexpr size_t thermodynamic_dim = 3;
```

Comment:
this is the number of independent thermodynamic quantities in the EOS, not the spatial dim.  We currently don't have any 3-dim EOS, (e,g, 1d polytropic fluid, 2d ideal fluid)

Reply:
Ah yes, for neutrinos, we need a temperature AND composition dependent EOS, so 3D... but maybe I shouldn't require that yet, it won't matter for the early tests.